app/controllers/course/users_controller.rb

```diff
@@ -45,6 +46,18 @@ def course_user_params # :nodoc:
                             permit(:user_id, :name, :workflow_state, :role, :phantom)
   end
 
+  def load_resource
```

Comment:
There was a duplicate rule that prevented the other rule which uses these attributes from triggering the bullet warning.